src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/MetaUtil.java

```diff
@@ -283,4 +281,11 @@ public static String identityHashCodeString(Object obj) {
         }
         return obj.getClass().getName() + "@" + System.identityHashCode(obj);
     }
+
+    /**
+     * Creates an immutable list from a trusted array.
```

Comment:
Can you please spell out what "trusted" means here (it's a shame JavaUtilCollectionAccess doesn't).